packages/shell-api/src/database.ts

```diff
@@ -234,7 +233,7 @@ export default class Database extends ShellApiClass {
    */
   @returnsPromise
   async runCommand(cmd: Document): Promise<Document> {
-    assertArgsDefined(cmd);
+    assertArgsDefinedType([cmd], [true], 'Database.runCommand');
```

Comment:
Not sure if we want to do something about it in the scope of this ticket, but we don't have any validation if input is a document and this leads to some very weird results when I'm providing non document input:

<img width="833" alt="image" src="https://user-images.githubusercontent.com/5036933/112291503-7588a200-8c90-11eb-92fe-e5d7ec678300.png">

Reply:
Yeah for these cases I did not start to change even more but I tried to just "replace" where I could immediately infer the logic. But in these cases it would probably make sense to check for `typeof cmd === 'object'`.